lms/templates/edxnotes/toggle_notes.html

```diff
@@ -8,12 +8,13 @@
   edxnotes_visibility_url = reverse("edxnotes_visibility", kwargs={"course_id": course.id})
 %>
 <div class="wrapper-utility edx-notes-visibility">
+  <span class="action-toggle-message" aria-hidden="true">${_("Hiding Notes")}</span>
```

Comment:
Don't aria-hide anything here.  Let it all out. aria-live is a very elegant solution since you don't want to encumber the screen reader user by moving their focus around to the message.  This informs them what just happened when its convenient for them and doesn't require them to tab, close, accept, etc. to dismiss it or move on.  elegant...